setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='1.1.10',
+    version='1.1.11',
```

Comment:
@efischer19 go ahead and start the release process-- I did merge something on Friday, so I will have something to verify.